android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetMediaProjectionService.java

```diff
@@ -44,7 +43,7 @@ public class JitsiMeetMediaProjectionService extends Service {
     static final int NOTIFICATION_ID = new Random().nextInt(99999) + 10000;
 
     public static void launch(Context context) {
-        OngoingNotification.createOngoingConferenceNotificationChannel();
+        NotificationUtils.createNotificationChannel("");
```

Comment:
Use `ongoing_notification_channel_name`, multiple notifications can go in the same channel.